desafio-03/valeriasouza28/javascript/palindromo.js

```diff
@@ -0,0 +1,20 @@
+const readline = require("readline-sync");
+const numbers = [];
+const palindromeNumbers = [];
+const start = readline.question("Digite um número inicial: ");
+const end = readline.question("Digite um número final: ");
+
+for (let i = start; i <= end; i++) {
+    numbers.push(i.toString());
```

Comment:
Aqui poderia salvar o número em uma constante ao invés de um array. E utilizar toda a lógica do segundo for neste mesmo loop.